apps/heft/src/plugins/CopyFilesPlugin.ts

```diff
@@ -218,7 +211,9 @@ export default class CopyFilesPlugin implements IHeftTaskPlugin<ICopyFilesPlugin
     taskSession.hooks.registerFileOperations.tap(
       PLUGIN_NAME,
       (operations: IHeftTaskFileOperations): IHeftTaskFileOperations => {
-        for (const operation of _resolveCopyOperationPaths(heftConfiguration, pluginOptions.copyOperations)) {
+        for (const operation of pluginOptions.copyOperations) {
+          // Do this here so that we only have to do it once
+          normalizeCopyOperation(heftConfiguration.buildFolderPath, operation);
```

Comment:
The result of this hook evaluation is passed to the `normalizeCopyOperation` loop in `TaskOperationRunner`, so converting here is redundant. You've merged the `_resolveCopyOperationPaths` with `normalizeFileGlobSpecifier` in `normalizeCopyOperation`, so this call and the call in `TaskOperationRunner` now each contain both halves of the normalization.